advanced/sr_gui_cyberglove_calibrator/src/sr_gui_cyberglove_calibrator/cyberglove_calibrator.py

```diff
@@ -366,29 +366,28 @@ def calibrate_current_step(self):
             self.btn_save.setEnabled(True)
 
     def save_calib(self):
-        filename = QtGui.QFileDialog.getSaveFileName(
+        filename = QtWidgets.QFileDialog.getSaveFileName(
             self._widget, 'Save Calibration', '')
         if filename == "":
             return
 
         self.calibrer.write_calibration_file(filename)
 
         # QMessageBox returns 0 for yes
-        if QtGui.QMessageBox.information(self._widget,
-                                         "Load new Calibration",
-                                         "Do you want to load the new calibration file?",
-                                         "yes",
-                                         "no") == 0:
+        if QtWidgets.QMessageBox.information(self._widget,
+                                             "Load new Calibration",
+                                             "Do you want to load the new calibration file?",
+                                             "yes",
+                                             "no") == 0:
             self.load_calib(filename)
 
     def load_calib(self, filename=""):
         if "" == filename:
-            filename = QtGui.QFileDialog.getOpenFileName(
+            filename = QtWidgets.QFileDialog.getOpenFileName(
                 self._widget, 'Open Calibration', '')
             if "" == filename:
                 return
 
         self.calibrer.load_calib(str(filename))
 
-        self.emit(QtCore.SIGNAL("messageToStatusbar(QString)"),
-                  "New Cyberglove Calibration Loaded.")
+        self.messageToStatusbar['QString'].emit("New Cyberglove Calibration Loaded.")
```

Comment:
probably this is bag that still exists. Actually when I was testing it, I launched the glove with no error but I was getting an error from rqt that there is no glove.